Testreglar/2.4.1/Nett/nett-2.4.1a.json

```diff
@@ -0,0 +1,501 @@
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.12",
+					"delutfall": {
+						"nr": 5,
+						"fasit": "Ja",
+						"tekst": "<br>- ligger før blokken med innhold"
+					}
+				},
+				"nei": {
+					"type": "regler",
+					"regler": {
+						"1": {
+							"type": "lik",
+							"sjekk": "3.3",
+							"verdi": "Ja",
+							"handling": {
+								"type": "regler",
+								"regler": {
+									"1": {
+										"type": "talDersom",
+										"sjekk": [
+											"3.4",
+											"3.5",
+											"3.6",
+											"3.8",
+											"3.9"
+										],
+										"verdi": "Ja",
+										"mellom1": 0,
+										"mellom2": 4,
+										"handling": {
+											"type": "avslutt",
+											"fasit": "Nei",
+											"utfall": "Det finnes en snarveilenke for å hoppe til hovedinnholdet, der snarveilenken#delutfall(0)#delutfall(1)#delutfall(2)#delutfall(3)#delutfall(4)<br><br>Det finnes en mekanisme for å slå sammen det gjentatte innholdet for å hoppe til hovedinnholdet, der mekanismen<br>- ikke ligger før blokken med innhold"
+										}
+									}
+								}
+							}
+						}
+					}
+				}
+			}
+		},
+		{
+			"stegnr": "3.12",
+			"spm": "Er mekanismen for å slå sammen, alltid synlig eller blir den synlig ved fokus?",
+			"ht": "<p>Mekanismen kan enten alltid være synlig, eller bli synlig ved tastaturfokus.</p>",
+			"type": "jaNei",
+			"kilde": [
+				"G1",
+				"G123"
+			],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.13",
+					"delutfall": {
+						"nr": 6,
+						"fasit": "Ja",
+						"tekst": "<br>- alltid er synlig eller blir synlig ved fokus"
+					}
+				},
+				"nei": {
+					"type": "regler",
+					"regler": {
+						"1": {
+							"type": "lik",
+							"sjekk": "3.3",
+							"verdi": "Ja",
+							"handling": {
+								"type": "regler",
+								"regler": {
+									"1": {
+										"type": "talDersom",
+										"sjekk": [
+											"3.4",
+											"3.5",
+											"3.6",
+											"3.8",
+											"3.9"
+										],
+										"verdi": "Ja",
+										"mellom1": 0,
+										"mellom2": 4,
+										"handling": {
+											"type": "avslutt",
+											"fasit": "Nei",
+											"utfall": "Det finnes en snarveilenke for å hoppe til hovedinnholdet, der snarveilenken#delutfall(0)#delutfall(1)#delutfall(2)#delutfall(3)#delutfall(4)<br><br>Det finnes en mekanisme for å slå sammen det gjentatte innholdet for å hoppe til hovedinnholdet, der mekanismen<br>- ligger før blokken med innhold<br>- ikke alltid er synlig eller blir synlig ved fokus"
+										}
+									}
+								}
+							}
+						}
+					}
+				}
+			}
+		},
+		{
+			"stegnr": "3.13",
+			"spm": "Slå sammen blokken med det gjentatte innholdet.",
+			"ht": "",
+			"type": "instruksjon",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "3.14"
+				}
+			}
+		},
+		{
+			"stegnr": "3.14",
+			"spm": "Blir tastaturfokus flyttet til det første aktive elementet i hovedinnholdet?",
```

Comment:
Har ikke du 5-steg for å komme til hovedinnholdet ref steg 3.10?

Reply:
Nei, det er ikke det samme.